web_listview_custom_column/models/ir_ui_view.py

```diff
@@ -39,6 +39,21 @@ def custom_column(self, diff):
                     element.getnext().addnext(element)
                     break
         elif diff['operation'] == 'reset':
+            # if empty, then search if a global view has been set on this
+            # and prompt user who tries to delete to talk to user who
+            # created those custom views.
+            if not customized_view:
+                views = self.search([
+                    ('model', '=', self.model),
+                    ('mode', '=', 'extension'),
+                    ('user_ids', '=', False),
+                    ('create_uid', '!=', self.env.user.id,),
```

Comment:
no trailing commas in tuples with len>1: https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas